CHANGELOG.md

```diff
@@ -5,6 +5,18 @@
 
 # Latest
 * Bumped xamlTools to 17.13.35422.31 (PR: [#7685](https://github.com/dotnet/vscode-csharp/pull/7685))
+* Update razor to 9.0.0-preview.24524.4 (PR: [#7692](https://github.com/dotnet/vscode-csharp/pull/7692))
+  * Roslyn Tokenizer (#11086) (PR: [#11086](https://github.com/dotnet/razor/pull/11086))
+    * **Experimental feature**
+  * Add flush method and make it implementation detail on how that happens (#11087) (PR: [#11087](https://github.com/dotnet/razor/pull/11087))
+  * Do not extract component into code block (#11069) (PR: [#11069](https://github.com/dotnet/razor/pull/11069))
+    * **New code action to handle extracting razor code into a new razor component**
+  * Handle EditorRequired *Changed/*Expression parameters (#11043) (PR: [#11043](https://github.com/dotnet/razor/pull/11043))
+  * Avoid ambiguous `object` reference in generic component recovery (#11053) (PR: [#11053](https://github.com/dotnet/razor/pull/11053))
+  * Move culture info check (#11057) (PR: [#11057](https://github.com/dotnet/razor/pull/11057))
+  * Report a better error for void components (#11041) (PR: [#11041](https://github.com/dotnet/razor/pull/11041))
+  * Ensure model directives are mapped at runtime (#11007) (PR: [#11007](https://github.com/dotnet/razor/pull/11007))
+  * Including @using for Out-of-Scope Razor Component References (#10651) (PR: [#10651](https://github.com/dotnet/razor/pull/10651))
```

Comment:
This is an odd one. Perhaps replace the "Do not extract component into code block" item with a link to https://github.com/dotnet/razor/pull/11019 since that bug never made it to VS Code anyway?

Reply:
yea, I guess I could just add notes about general availability of it without a PR?

Reply:
Yeah, whatever works